cassandra.go

```diff
@@ -75,6 +75,20 @@ func Open(rawurl string) (driver.Driver, error) {
 		}
 
 	}
+	// handle ssl option
+	if len(u.Query().Get("sslmode")) > 0 {
+		if u.Query().Get("sslmode") != "disable" {
+			cluster.SslOpts = &gocql.SslOptions{}
+			if len(u.Query().Get("sslrootcert")) > 0 && len(u.Query().Get("sslcert")) > 0 && len(u.Query().Get("sslkey")) > 0 {
```

Comment:
At this point (`sslmode` has been tested), I think it's safe to assign these values directly.
They will default to zero values anyway. What do you think?